boards/arm/mimxrt1010_evk/mimxrt1010_evk.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: mimxrt1010_evk
+name: NXP MIMXRT1010-EVK
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 32
+flash: 16384
```

Comment:
Make this 32. This should help resolve the shippable failures

Reply:
Sure, thanks.